src/maxPriorityQueue.js

```diff
@@ -13,9 +13,7 @@ const { PriorityQueue } = require('./priorityQueue');
 class MaxPriorityQueue extends PriorityQueue {
   constructor(options) {
     super(options);
-    if (this._compare) {
-      this._heap = new CustomHeap(this._compare);
-    } else {
+    if (!this._compare) {
```

Comment:
Won't change in children for backward compatibility.

Reply:
Since both `MaxPriorityQueue` and `MinPriorityQueue` are inherited `PriorityQueue`, it'll use the customHeap as default, so for the `_compare` case we don't need to apply the customHeap again

https://github.com/datastructures-js/priority-queue/pull/30/files#diff-486eb363866f5bf49b111abd172862b5746d6da76e55ae01fb3e5ca3f316fd3fR23